src/CalcManager/CalculatorVector.h

```diff
@@ -120,12 +114,12 @@ class CalculatorVector
 
     ResultCode GetString(_Out_ std::wstring* expression)
     {
-        ResultCode hr = S_OK;
         unsigned int nTokens = 0;
-        std::pair<std::wstring, int> currentPair;
-        hr = this->GetSize(&nTokens);
+        auto hr = this->GetSize(&nTokens);
```

Comment:
```suggestion
        ResultCode hr = this->GetSize(&nTokens);
```

I don't see any benefit to using `auto` in this scenario.